packages/cpp/src/common/cpp-preferences.ts

```diff
@@ -14,32 +14,38 @@ import {
     PreferenceSchema
 } from '@theia/preferences/lib/common';
 
+export const CLANGD_COMMAND_DEFAULT = 'clangd';
+
 export const CppConfigSchema: PreferenceSchema = {
     "type": "object",
     "properties": {
-        "cpp.clangdCompileCommandsPath": {
+        "cpp.clangdCompilationDatabaseDirectory": {
             "type": "string",
-            "description": "clangd path for compile_commands.json"
+            "description": "Path to the Compilation Database directory (containing compile_commands.json). " +
+                "This file contains compiler commands for each source files which are important for Clangd to properly analyze files. " +
+                "It can be generated by several tools such as CMake."
         },
-        "cpp.clangdPath": {
+        "cpp.clangdCommand": {
             "type": "string",
-            "description": "Literal command to start Clangd with. Can contain any relative or absolute path to clangd executable."
+            "description": "Command to launch the clangd executable."
         },
         "cpp.clangdCommandArgs": {
-            "type": "string",
-            "description": "Literal command to start Clangd with. Can contain any relative or absolute path to clangd executable."
+            "type": "array",
+            "description": "Arguments to the Clangd command."
```

Comment:
Here, the `Clangd` is with capital, above it is with lower case. Which one is the truth?

Reply:
Clangd is the name of the tool, clangd is the executable. Probably it's clearer to use clangd in both places in this case.

Reply:
Oh, I see. Thanks for explaining it.